recipes/z5py/build.sh

```diff
@@ -0,0 +1,67 @@
+# Platform-specific dylib extension
+if [ $(uname) == "Darwin" ]; then
+    export CC=clang
+    export CXX=clang++
+    export DYLIB="dylib"
+else
+    export CC=gcc
+    export CXX=g++
+    export DYLIB="so"
+fi
```

Comment:
This section can be dropped as the compiler syntax (explained more fully below) takes care of this. Also the `CXXFLAGS` and `LDFLAGS` below can be dropped as well for the same reason.

Also `conda-build` sets [`SHLIB_EXT`]( https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html#environment-variables-set-during-the-build-process ), which takes the place of `DYLIB`. Would add `SHLIB_EXT` includes the `.`.